README.md

```diff
@@ -93,6 +96,23 @@ Hello, VS Code Remote - Containers!
 {"outcome":"success"}
 ```
 
+Congrats, you've just run the dev container CLI and seen it in action!
+
+## Build from sources
```

Comment:
Would we want this section before `Try out the CLI`, so that users know they can try it with the npm package or building from sources?

Reply:
The same README is used for the package on npmjs.org as with the repository on github.com. Not sure if there is a way to have distinct READMEs for the two. Giving the use on npmjs.org priority for now.